src/Test/Unit/Config/Validator/Deploy/SearchConfigurationTest.php

```diff
@@ -0,0 +1,117 @@
+<?php
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+namespace Magento\MagentoCloud\Test\Unit\Config\Validator\Deploy;
```

Comment:
Add `strict_types` declaration to all new classes

Reply:
Done